### PR TITLE
RT-511338 added new featured topic to the homepage

### PR DESCRIPTION
**Pull request by MGS-sails**
…511338

### CODE REVIEW

File: src/site/index.njk [hunk 1]
```diff
@@ -166,18 +166,21 @@ templateEngineOverride: njk
         <div class="medium-8 columns | analytics-content-left | sticky-feature">
           <h3 class="vf-section-header__heading"><a href="https://www.covid19dataportal.org/support-data-sharing-covid19" class="no-underline">Featured topic</a></h3>
           <div class="vf-u-background-color--bright-green--dark padding-bottom-large margin-top-small" style="background: rgb(59,73,80);">
-              <a href="https://www.surveymonkey.co.uk/r/EMBL-EBI_Impact"><img src="https://acxngcvroo.cloudimg.io/v7/https://www.embl.org/files/wp-content/uploads/Covid19_Data_Portal_screenshot.png" alt="Our coronavirus response" /></a>
+            <a href="https://www.surveymonkey.co.uk/r/EMBL-EBI_Impact">
```
Comment (sandykadam) [inline src/site/index.njk:169]: Update this link to the button link mentioned

File: src/site/index.njk [hunk 2]
```diff
@@ -166,18 +166,21 @@ templateEngineOverride: njk
...
+              <div class='embed-container vf-card__image'><iframe src='https://www.youtube.com/embed/RrrSoW5euX0'
+                                          frameborder='0' allowfullscreen></iframe></div>
+            </a>
             <div class="vf-u-text-color--ui--white">
               <div class="vf-u-margin__top--600 vf-u-margin__right--800 vf-u-margin__left--800 ">
                 <p class="vf-text--body vf-text-body--1">
-                  <a href="https://www.surveymonkey.co.uk/r/EMBL-EBI_Impact" class="vf-link--white no-underline">Our coronavirus response</a>
+                  <a href="https://www.surveymonkey.co.uk/r/EMBL-EBI_Impact" class="vf-link--white no-underline">MGnify: Harnessing nature’s bioengineering</a>
                 </p>
```
Comment (sandykadam) [inline src/site/index.njk:177]: Same here update the link to the button link mentioned

File: src/site/index.njk (hunk 1 above)
Comment (MGS-sails) [reply src/site/index.njk:169]: I think we should remove the href here, as it's now an embedded video being shown instead of an image

Comment (sandykadam) [reply src/site/index.njk:169]: Yes we can do that as well

File: src/site/index.njk (hunk 2 above)
Comment (MGS-sails) [reply src/site/index.njk:177]: Oana's mentioned that this should link to the video, not the services page. I'll update the ticket with this info

**Pull request merged**